soccer/Robot.hpp

```diff
@@ -530,6 +532,9 @@ class OurRobot : public Robot {
     friend class MotionControl;
 
 private:
+    mutable RJ::Time _lastBallSense;
```

Comment:
Is there a good way to check for ball sense information without having to call hasball every frame?